src/libbson/fuzz/fuzz_test_bson_utf8_escape_for_json.c

```diff
@@ -0,0 +1,16 @@
+#include <bson/bson.h>
+
+int
+LLVMFuzzerTestOneInput (const uint8_t *data, size_t size)
+{
```

Comment:
```suggestion
{
   if (bson_cmp_greater_us (size, SSIZE_MAX)) {
      return -1;
   }

```
Reject inputs whose length exceeds `ssize_t`.